library.json

```diff
@@ -13,7 +13,7 @@
         "url": "https://github.com/hideakitai",
         "maintainer": true
     },
-    "version": "0.1.0",
+    "version": "0.1.1",
```

Comment:
You don't need to change the version, I will do it later

Reply:
That came from your branch.